src/loader.js

```diff
@@ -30,7 +38,15 @@ function hotLoader(content, context) {
     ...context.options,
     locals: !!context.locals,
   })});
-      module.hot.dispose(cssReload);
+      var _locals = ${JSON.stringify(context.locals)};
+      var isEqualLocals = require(${loaderUtils.stringifyRequest(
```

Comment:
When `esModule` is true it should be `import`,

Reply:
but previously it has 
```js
var cssReload = require(${loaderUtils.stringifyRequest(
        context.context,
        path.join(__dirname, 'hmr/hotModuleReplacement.js')
```
This should be import too?

Reply:
Yep

Reply:
fixed